pyanaconda/modules/common/__init__.py

```diff
@@ -15,21 +15,40 @@
 # License and may only be used or replicated with the express permission of
 # Red Hat, Inc.
 #
+import sys
 
 __all__ = ["init"]
 
 
-def init():
+def init(filename=None, stream=sys.stderr):
```

Comment:
I think calling those two `log_filename` & `log_stream` would be better - even though both are documented as log related, it would be good to have this also in the name for parameters of such a top level function.

Reply:
Fixed.